src/graphql/mutations/update-online-user.ts

```diff
@@ -0,0 +1,24 @@
+import { useMutation } from 'graphql-hooks';
+
+const MUTATION = /* GraphQL */ `
+  mutation UpdateUser($user: String!) {
+    update_user(where: { id: { _eq: $user } }, _set: { last_seen: "now()" }) {
```

Comment:
didn't know about this "now()" parameter :-)

Reply:
Yes I can rebase on top master. Sure!